cloud-deployment-guide/editor-plugin-version.md

```diff
@@ -20,308 +20,117 @@ This URL specifies the latest and quality assured release of {{site.productname}
 
 ### Selecting specific version numbers
 
-> All {{site.cloudname}} channels are based on the {{site.enterpriseversion}} version. For information on the latest version of the {{site.cloudname}} stable channel, see: [{{site.productname}} Release Notes]({{site.baseurl}}/release-notes/). For a list of changes that **may** be present in the {{site.cloudname}} testing channel, see: [{{site.productname}} Changelog]({{site.baseurl}}/changelog/).
+> All {{site.cloudname}} channels are based on the {{site.enterpriseversion}} version. For information on the latest version of the {{site.cloudname}} {{site.productmajorversion}} channel, see: [{{site.productname}} Release Notes]({{site.baseurl}}/release-notes/). For a list of changes that **may** be present in the {{site.cloudname}} testing channel, see: [{{site.productname}} Changelog]({{site.baseurl}}/changelog/).
 
-Support for requesting specific versions of {{site.productname}} {{site.productmajorversion}} should work same as {{site.productname}} 4.  In the meantime, there are three release channels available, see the section below.
+### {{site.productmajorversion}}-dev, {{site.productmajorversion}}-testing, and {{site.productmajorversion}} releases
```

Comment:
Do these version numbers need to be in `quotes` like they are on the headings below?

Reply:
it is on the headings for examples, but not on the others. so, not necessary at this time?

Reply:
Oh, I definitely wasn't consistent. I figured someone would cleanup after me ...